test/solvers/functional/functional_test.py

```diff
@@ -280,11 +280,9 @@ def test_functional_sum(space):
                             func2.gradient(x).inner(p)),
                         places=PLACES)
 
-    # Test that prox and convex conjugate is not known
+    # Test that prox is not known
     with pytest.raises(NotImplementedError):
         (func1 + func2).proximal
-    with pytest.raises(NotImplementedError):
```

Comment:
The thing is that with the default convex conjugate added, this line won't raise any more but rather return an object that cannot be evaluated, but still asked for its convex conjugate and proximal.